doc/release/README.chplenv.rst

```diff
@@ -0,0 +1,555 @@
+   listed above, please refer to the README.building file and/or contact us at:
+   chapel_info@cray.com
+
+#. Set your PATH to include the directory $CHPL_HOME/bin/$CHPL_HOST_PLATFORM
+   which is created when you build the compiler.  For example::
+
+        export PATH="$PATH":"$CHPL_HOME/bin/$CHPL_HOST_PLATFORM"
+
+
+#. Set your man path to include the directory $CHPL_HOME/man.
+   For example::
+
+        export MANPATH="$MANPATH":"$CHPL_HOME"/man
+
+Optional Settings
+-----------------
+
+CHPL_HOST_PLATFORM
+~~~~~~~~~~~~~~~~~~
+   If you are cross-compiling for a platform other than your
+   $CHPL_HOST_PLATFORM, set the CHPL_TARGET_PLATFORM environment variable to
+   describe that platform.  See CHPL_HOST_PLATFORM above for legal values
+   (though whether or not a given setting will support cross-compilation
+   depends on your specific environment).
+
+   .. note::
+     If CHPL_TARGET_PLATFORM is not set, the target platform defaults to the
+     same value as $CHPL_HOST_PLATFORM.
+
+CHPL_*_COMPILER
+~~~~~~~~~~~~~~~
+   Optionally, you can set CHPL_HOST_COMPILER and/or
+   CHPL_TARGET_COMPILER to indicate the compiler suite to use in building the
+   sources.  CHPL_HOST_COMPILER is the compiler used to build the Chapel
+   compiler itself so that it will run on CHPL_HOST_PLATFORM.
+   CHPL_TARGET_COMPILER is the compiler used to build the runtime libraries and
+   generated code for CHPL_TARGET_PLATFORM.  Currently supported values are as
+   follows:
+
+        =================  ===================================================
+        Value              Description
+        =================  ===================================================
+        clang              The Clang compiler suite -- clang and clang++
+        clang-included     The Clang compiler in third-party/llvm
+        cray-prgenv-cray   The Cray PrgEnv compiler using the Cray CCE backend
+        cray-prgenv-gnu    The Cray PrgEnv compiler using the GNU backend
+        cray-prgenv-intel  The Cray PrgEnv compiler using the Intel backend
+        cray-prgenv-pgi    The Cray PrgEnv compiler using the PGI backend
+        gnu                The GNU compiler suite -- gcc and g++
+        ibm                The IBM compiler suite -- xlc and xlC
+        intel              The Intel compiler suite -- icc and icpc
+        pgi                The PGI compiler suite -- pgcc and pgCC
+        =================  ===================================================
+
+   The default for CHPL_*_COMPILER depends on the value of the corresponding
+   CHPL_*_PLATFORM environment variable:
+
+        ============  ==================================================
+        Platform      Compiler
+        ============  ==================================================
+        cray-x*       - gnu (for CHPL_HOST_COMPILER)
+                      - cray-prgenv-$PE_ENV (for CHPL_TARGET_COMPILER,
+                        where PE_ENV is set by PrgEnv-* modules)
```

Comment:
Yes, in the html output there is no word wrap and the user is required to scroll to the right in order to read the full text.